scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
+
+However, the SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a
+minimum. On a normal ethernet network, this should be set to around 1450. On a
+jumbo frame ethernet network, this should be around 8950.
+
+To configure the MTU, edit the node configuration file at *_/etc/origin/node/node-config.yaml_* and edit the following:
```

Comment:
Yeah, it's best to set the MTU at cluster standup time rather than adjust it later; you'd need to take the cluster down and then back up to ensure that you don't have glitches related to fragmentation and the MTU.

@knobunc your comment only applies to OpenShift SDN I think, right?